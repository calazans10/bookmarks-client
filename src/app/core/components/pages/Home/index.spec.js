import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { Home } from './index';

describe('Home', () => {
  it('renders without crashing', () => {
    const props = {
      onRequestLoading: jest.fn(),
    };
    const wrapper = shallow(<Home {...props} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
