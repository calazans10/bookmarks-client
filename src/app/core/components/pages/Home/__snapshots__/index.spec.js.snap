// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders without crashing 1`] = `
<MainWrapper>
  <MainHeader>
    <Logo
      to="/"
    />
    <Connect(MainNavigation)
      pathname="/sign_in"
      title="Sign In"
    />
  </MainHeader>
  <MainContent>
    <PageSection
      title="A smarter way to save your web pages"
    >
      <p>
        Bookmarks makes it simple to save web pages you find useful, interesting, or inspiring in a beautiful, easy-to-use web app.
      </p>
      <p>
        Don't have an account? 
        <Link
          to="/sign_up"
        >
          Sign up here
        </Link>
        .
      </p>
    </PageSection>
  </MainContent>
</MainWrapper>
`;
